modules/tinymce/src/core/main/ts/api/Settings.ts

```diff
@@ -104,7 +104,7 @@ const shouldIndentUseMargin = (editor: Editor): boolean => editor.getParam('inde
 const getIndentation = (editor: Editor): string => editor.getParam('indentation', '40px', 'string');
```

Comment:
It would be nice to give more of these `getParam` calls default values and types... That is outside the scope of this Jira though, so maybe we should make another Jira for it? Hm

Reply:
I tried, but I got a bit worried since I didn't know what the default value would be, and didn't want to put something to default to true in one plugin, when it should be false.

Reply:
Yeah, I wouldn't do it atm, as there's a fair bit of risk involved in that. So I'd also suggest just logging a follow-up JIRA like @metricjs suggested.

Reply:
Added JIRA as requested.